inputFiles/poromechanics/PoroElastic_staircase_co2_3d_sequential.xml

```diff
@@ -0,0 +1,398 @@
+<?xml version="1.0" ?>
```

Comment:
Is it possible to put some of these blocks in a base XML file to avoid redundancy?

Reply:
yes i will definitely do that before merging, thanks!

Reply:
done